demos/TransferLearning/TransferLearningDemo.py

```diff
@@ -1,11 +1,10 @@
 import warnings
 warnings.simplefilter('ignore')
-
-import argparse
-
 import ngraph_bridge
-ngraph_bridge.enable()
 
+import argparse
+import sys
+import importlib
 import numpy as np
 import matplotlib.pyplot as plt
 import matplotlib.image as mpimg
```

Comment:
VSCode's python plugin has a nice feature that sorts imports. It will put python modules first, then third party next.